mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
+plugins = pydantic.mypy
```

Comment:
Why is it here ?

Reply:
Long-story-short: mypy opines on some specific points (in that case, class __init__) that causes it to misinterpret some of pydantic behaviours, causing errors that are not true. Pydantic has a mypy plugin to fix that.

Reply:
This should global and not per project

Reply:
Didn't get it.

Reply:
We should avoid personal specific files in the repository, such as IDE configuration files and etc.
If this file is related to mypy checks used in CI, it should exist in the global folder of this project and not in this service only.

Reply:
But it is in the global folder.

Reply:
Right, sorry for that.
We still need to figure a better way to use that, maybe with the hook script ?
Right now mypy does not support pyproject, so I'm afraid that this project ends up with a bunch of files in the global folder for python tools configuration.
@Williangalvani any tip ?

Reply:
Also, should we add such plugin in our CI to use it ?

Reply:
> Right, sorry for that.
> We still need to figure a better way to use that, maybe with the hook script ?
> Right now mypy does not support pyproject, so I'm afraid that this project ends up with a bunch of files in the global folder for python tools configuration.
> @Williangalvani any tip ?

No prob.
As I understand, mypy is probably one of the few, if not the only, tool that [does not support or plan to support pyproject.toml](https://github.com/python/mypy/issues/5205), so we can have only pyproject.toml and mypy.ini.

> Also, should we add such plugin in our CI to use it ?

No need. Pydantic is already a sub-dependency of fastapi.